ec/acc/testdata/deployment_basic_defaults_3.tf

```diff
@@ -12,6 +12,7 @@ resource "ec_deployment" "defaults" {
   elasticsearch {
     topology {
       size = "1g"
+      node_type_ingest = "false"
```

Comment:
@karencfv any reason on why we opt using strings here instead of boolean?

Reply:
Because booleans in Go default to false instead of nil. 
If we were to keep booleans, when a user wanted to use the default node types they would leave it blank and it would set to false. With strings we can check for an empty string and fill in with defaults if the string is empty.